app/controller/sdl/RCModulesController.js

```diff
@@ -329,10 +338,44 @@ SDL.RCModulesController = Em.Object.create({
               }
             };
             windowStatus.push(location_window_status);
+
+            var door_status = {
+              "location": seat_location,
+              "status": "CLOSED"
+            }
+            doorStatuses.push(door_status);
           }
         });
 
         SDL.SDLVehicleInfoModel.vehicleData.windowStatus = windowStatus;
+        SDL.SDLVehicleInfoModel.vehicleData.bodyInformation.doorStatuses = doorStatuses;
```

Comment:
@LitvinenkoIra suggest to move this line to the rest of statuses at line 377 to have just one place for all of them